domains/gav.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jgavinb",
+        "email": "jw442501@gmail.com"
+    },
+    "record": {
+        "CNAME": "jgavinb.github.io"
```

Comment:
I'm unable to view this site or see a valid GitHub repository. Remember GitHub Pages must be public in order for it to be accessible.